temfpy/integration_methods.py

```diff
@@ -0,0 +1,169 @@
+"""Integration Methods.
+We provide a variety of numerical integration methods.

```

Comment:
No, but I was not sure if we will do one at some point. Hence, I thought this would facilitate further implementations. I think we can leave it and see if we can use it in the future?